src/supervisor.adoc

```diff
@@ -1265,6 +1296,9 @@ supervisor software does not rely on accessed and/or dirty bits, e.g. if
 it does not swap memory pages to secondary storage or if the pages are
 being used to map I/O space, it should always set them to 1 in the PTE
 to improve performance.
+
+The PTE updates due to memory accesses ordered-after a FENCE are not
+themselves ordered by the FENCE.
```

Comment:
This sentence appears twice in the same NOTE.  Can you delete one of them?

Reply:
done